### ISSUE TITLE
Exploding losses during voice-conversion training

### ISSUE THREAD

**Issue by neuronx1** [ebadawy/voice_conversion#17]
Thanks for the great repository.
Unfortunatly I have a problem during the voice-conversion training.

After the first 2 Epochs I get exploding losses.
![grafik](https://user-images.githubusercontent.com/62887153/149625688-e79a2ea7-53d4-43ad-801a-8de811c114e3.png)

What's the reason for that and how can I solve this?

I am happy about every tip

Thanks in advance!

**Comment by ebadawy** [comment]
@neuronx1 do you still have the problem? If possible can you mention what datasets did you use and the steps for setup/training process?

**Comment by neuronx1** [comment]
@ebadawy thanks for your reply!
Unfortunatley I still have the problem.
I used for training your pretrained model and finetuned this on my own dataset.
The dataset contains 2 speaker.
I have 900 audio files of each speaker, each between 5 and 30 seconds long.
In preprocessing, I have removed the silence from each of these files. The sampling rate is 22,050 Hz and the audio channel is mono. Also, the file is 16 bit.
I would be very happy to receive any tips!

**Comment by ebadawy** [comment]
did you use the same preprocessing that we do? it should be done through `src/preprocess.py`. I believe the current sample rate we are using is 16kHz. Probably you will need to change the hyperparamters if you want to use a different sample rate. Once thing i would recommend to try is to train the model from scratch without pretrained models. the final quality might not be as good but you should not get exploding loss.

**Comment by WilledgeR** [comment]
I have same problem
dataset: Flickr
speakers: number 4, number 7
default options of repository.

**Comment by 03vmate** [comment]
652fa0bdc0f774a4bedc81b5a549bad1c2158f2a completely broke train.py, revert it, and fix indentations yourself, that will fix this issue

**Comment by profection** [comment]
same problem for me, here's the picture
![Screenshot 2022-07-11 191008](https://user-images.githubusercontent.com/29281295/178251753-726f7640-1984-45cd-a05f-41e29cc2dcba.jpg)
is this related to the lambda value?
can you paste what your train record? @ebadawy

**Comment by ebadawy** [comment]
So the current state of the code seems to be not working properly. Unfortunately, i won't have the time to see where the bug is. However, to get a working version you can revert back to be362f9102e01e9e173657c5b31650c7dcc3699d. This is the original training pipeline that I know for sure to work.

Same solution should work for issues #20 #19 #15
